.github/workflows/spark-ci.yml

```diff
@@ -101,8 +102,9 @@ jobs:
   spark-3x-scala-2-13-tests:
     runs-on: ubuntu-22.04
     strategy:
+      fail-fast: false
       matrix:
-        jvm: [8, 11]
+        jvm: [8, 11, 17]
```

Comment:
we have java17 with tests spark3 in a seperate job below : 
` spark-3x-java-17-tests:`

Reply:
what is the reason for `spark-3x-java-17-tests` being separate?

Reply:
dropped